.drone.yml

```diff
@@ -1308,16 +1308,33 @@ steps:
       - cd /go/src/github.com/gravitational/teleport
       - git clone https://github.com/gravitational/${DRONE_REPO_NAME}.git .
       - git checkout ${DRONE_COMMIT}
+      - mkdir -p /go/chart
+      - cd /go/chart
+
+  - name: Download chart repo contents
+    image: amazon/aws-cli
+    environment:
+      AWS_S3_BUCKET:
+        from_secret: PRODUCTION_CHARTS_AWS_S3_BUCKET
+      AWS_ACCESS_KEY_ID:
+        from_secret: PRODUCTION_CHARTS_AWS_ACCESS_KEY_ID
+      AWS_SECRET_ACCESS_KEY:
+        from_secret: PRODUCTION_CHARTS_AWS_SECRET_ACCESS_KEY
+    commands:
+    - mkdir -p /go/chart
```

Comment:
Really a nit, but the indentation here is not consistent with other steps where each `- xxx` item has 2 spaces before it.